features/docs/gegeven-stap-definities-expressief.feature

```diff
@@ -373,6 +373,43 @@ Scenario: persoon zonder partner is overleden
     |            | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                              | 9999,0,0,P,000000012,P1  |
     |            | overlijden   | INSERT INTO public.lo3_pl_overlijden(pl_id,volg_nr,overlijden_datum) VALUES($1,$2,$3)                                                                                                                           | 9999,0,gisteren - 2 jaar |
 
+Scenario: persoon met partner is overleden
+  Gegeven de persoon 'P1' met burgerservicenummer '000000012'
+  * is overleden
```

Comment:
Ik zou deze stap hernoemen naar `[aanduiding] is overleden`, conform afspraak gisteren en verplaatsen onder de gehuwd stap